classes/tegra-uefi-signing.bbclass

```diff
@@ -1,3 +1,5 @@
+inherit signing_server
```

Comment:
Rather than directly inheriting the class here, how about we add a variable, something like `TEGRA_SIGNING_CLASS`, defaulting to the null string somewhere appropriate (maybe in the `tegra-common.inc` include file) and use that variable here - `inherit ${TEGRA_SIGNING_CLASS}`?  

That would allow another layer, or distro, to add the needed overrides in one place, without having to put specifics about the digsigserver implementation here in the BSP layer.  The same could be done in other places where signing steps might need modification - optee, BUP generation, etc.  We could even have different variables for each type of signing, if that seems appropriate.

I'd really like to give people a way to insert whatever signing mechanism they need here at the BSP layer level, rather than tying things to digsigserver specifically.

Reply:
I like it...  I'll work on that.